config/default/patch_cert_manager_monitoring.yaml

```diff
@@ -0,0 +1,7 @@
+- op: replace
```

Comment:
Can you please rename this file to something of the format `*_patch.yaml` instead? This would help keep parity in naming with other patch yaml(s) in the same folder.

How about `cert_manager_monitoring_patch.yaml`?